### ISSUE TITLE
selectedIndex property is not running correctly

### ISSUE THREAD

**Issue by duynhanf** [rakuten-rex/select#20]
I have an sample in here
https://codesandbox.io/s/zealous-rain-2e7wq
in example 2 and 3 cannot choose the correct option

**Comment by claudio-moya** [comment]
@duynhanf thanx for address the issue and the example, we will take a look!

**Comment by Kosukeee** [comment]
@duynhanf ,
Thanks for notifying the issue.
I checked the attached codesandbox, however I couldn't find out the issue.
Attached my recording, please check it.
https://rak.box.com/s/0h4vsrg5albbczv3iluyrbaxo35xhpbl

**Comment by duynhanf** [comment]
@Kosukeee Thank you for checking.
In example 1, 2

```
<SelectLabelHelper
        id="age1"
        label="Label"
        helper="Helper Text"
        placeholder="Placeholder"
        options={[
          { text: "Option 1", value: "option_1" },
          { text: "Option 2", value: "option_2" },
          { text: "Option 3", value: "option_3" },
          { text: "Option 4", value: "option_4" }
        ]}
        htmlFor="age1"
        selectedIndex={0}
      />

      <SelectLabelHelper
        id="age2"
        label="Label"
        helper="Helper Text"
        placeholder="Placeholder"
        options={[
          { text: "Option 1", value: "option_1" },
          { text: "Option 2", value: "option_2" },
          { text: "Option 3", value: "option_3" },
          { text: "Option 4", value: "option_4" }
        ]}
        htmlFor="age2"
        selectedIndex={2}
      />
```
I specify the selectbox's default value with selectedIndex but when initializing it doesn't select that value.
Is that a misunderstanding of the usage of selectedIndex?

**Comment by Kosukeee** [comment]
@duynhanf ,
Yes, you are correct.
I can check the issues too.

Let me check component itself.

**Comment by Kosukeee** [comment]
@duynhanf ,
Fixed the issue and released patched version 1.4.1.
Could you please update @rakuten-rex/select to 1.4.1 and test it again.

It seems working correctly on my side.
<img width="675" alt="Screen Shot 2020-08-17 at 16 41 41" src="https://user-images.githubusercontent.com/6793504/90370590-99578a00-e0a8-11ea-8536-bde933341377.png">

Thanks.

**Comment by duynhanf** [comment]
@Kosukeee Thank you for fixing. It's OK then I will close this issue.
